src/v2/cookbook/using-axios-to-consume-apis.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Using Axios to Consume APIs
+type: cookbook
+order: 9
+---
+
+## Simple Example
+
+There are many times when building application for the web that you may want to consume and display data from an API. There are several ways to do so, but a very popular approach is to use [axios](https://github.com/axios/axios), a promise-based HTTP client.
+
+In this exercise, we'll use the [Coindesk API](https://www.coindesk.com/api/) to walk through displaying Bitcoin prices, updated every minute. First, we'd install axios with either npm/yarn or through a CDN link.
```

Comment:
Sorry I missed this in the last review batch. Shouldn't it be CoinDesk (capital D) instead?

Reply:
Totally :) I'll get that in